DESCRIPTION

```diff
@@ -16,9 +16,9 @@ Imports:
   ggrepel,
   jaspBase,
   jaspGraphs,
-  jfa (>= 0.5.0),
```

Comment:
You need to specify `jfa` under `Imports` and `Remotes`: https://cran.r-project.org/web/packages/devtools/vignettes/dependencies.html